CONTRIBUTING.md

```diff
@@ -310,8 +310,38 @@ Leave a comment in the PR and we'll help you out.
 
 ### Testing plots
 
-We use the [pytest-mpl](https://github.com/matplotlib/pytest-mpl) plug-in to test plot
-generating code.
+Writing an image-based test is only slightly more difficult than a simple test.
+The main consideration is that you must specify the "baseline" or reference
+image, and compare it with a "generated" or test image. This is handled using
```

Comment:
Added some notes here to CONTRIBUTING.md, adapted from https://matplotlib.org/3.3.1/devel/testing.html#writing-an-image-comparison-test.